wallet/wallet-cli-lib/src/commands/mod.rs

```diff
@@ -295,16 +295,13 @@ pub async fn handle_wallet_command(
             }
         }
 
-        WalletCommand::GenerateBlock {
-            reward_destination: _,
-            transactions,
-        } => {
-            let transactions =
-                transactions.map(|txs| txs.into_iter().map(HexEncoded::take).collect());
-            let block = rpc_client
-                .generate_block(GenerateBlockInputData::None, transactions)
+        WalletCommand::GenerateBlock => {
```

Comment:
Does the user have to keep calling this to create blocks one-by-one? I'm not done reviewing yet, but if yes, then we need some kind of an event loop that automates this. The user should start this once and then the blocks get generated automatically.

Reply:
I added the `GenerateBlocks` command which takes the number of blocks to generate.

Reply:
Please create a function for normal users like `start_staking`, which will perpetually run and keep creating blocks. This is how monero-simple-wallet works too, I believe, where you issue a command it'll just keep mining.

Reply:
`startstaking` and `stopstaking` commands added. The wallet-controller will now try to generate new blocks every second in the background.